frontend/js/src/utils/types.d.ts

```diff
@@ -657,6 +657,13 @@ declare type BrainzPlayerSettings = {
   >;
 };
 
+declare type FlairPreferences =
```

Comment:
I think this might be better handled using an enum like 
```
export enum UserFlair {
  Default = "default",
  Air = "air",
  ...
}
```

Not sure we want to export an enum from this file though, maybe needs to move to....Settings.tsx I guess?

Reply:
I don't think we should export the enum from setings.tsx....

This is just a placeholder for now. Since we've not touched upon the implementation of user flairs for now everywhere, we might need to save additional information.